sidechainnet/utils/minimizer.py

```diff
@@ -0,0 +1,269 @@
+"""Minimizes SCNProteins."""
```

Comment:
add some more descriptive notes/comments, express that this is research-level code